www-servers/nginx-unit/nginx-unit-1.10.0.ebuild

```diff
@@ -0,0 +1,67 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=(python2_7 python3_{5,6,7})
+
+inherit python-single-r1
+
+MY_P="unit-${PV}"
+DESCRIPTION="A dynamic web and application server"
+HOMEPAGE="https://unit.nginx.org"
+SRC_URI="https://unit.nginx.org/download/${MY_P}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="Apache-2.0"
+SLOT="0"
+KEYWORDS="~amd64"
+MY_USE="perl python ruby"
+MY_USE_PHP="php5-6 php7-1 php7-2 php7-3"
```

Comment:
Any chance of this working with php-7.4 already? There's a slot, albeit a beta version.

Reply:
Adding support in the ebuild would be easy enough. Alas, I don't have access to any Gentoo system running PHP 7.4 for testing. Should I expand the ebuild anyway?

Reply:
Naa, test it first :)